cleanlab/datalab/internal/issue_manager/null.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, ClassVar, Dict, Optional
+
+import numpy as np
+import pandas as pd
+
+from cleanlab.datalab.internal.issue_manager import IssueManager
+
+if TYPE_CHECKING:  # pragma: no cover
+    import numpy.typing as npt
+    from cleanlab.datalab.datalab import Datalab
+
+
+class NullIssueManager(IssueManager):
+    """Manages issues related to null/missing values in the rows of features.
+    Parameters
+    ----------
+    datalab :
+        The Datalab instance that this issue manager searches for issues in.
+    """
+
+    description: ClassVar[
+        str
+    ] = """Whether the dataset has any missing/null values
+        """
+    issue_name: ClassVar[str] = "null"
+    verbosity_levels = {
+        0: ["average_null_score"],
+        1: [],
+        2: [],
```

Comment:
I've suggested additional information to store. I suggest you include them at higher verbosity levels.

Say, both at level 1.